packages/forms/resources/views/components/placeholder.blade.php

```diff
@@ -15,7 +15,10 @@
         {{
             $attributes
                 ->merge($getExtraAttributes(), escape: false)
-                ->class(['fi-fo-placeholder sm:text-sm'])
+                ->class([
+                    'fi-fo-placeholder sm:text-sm',
+                    'py-1.5' => $hasInlineLabel() && !$getContent() instanceof \Illuminate\Contracts\Support\Htmlable
```

Comment:
Also prevent py-1.5 for Hmtlable content because sometimes we want to use border or different background for the content.